ProcessMaker/processmaker
# FOUR-14483: Navigate or edit Screen Template

## Issue & Reproduction Steps
Users are encountering an issue where they cannot view the correct Screen Templates in Screen Builder. After saving a Screen as a Template and attempting to edit it from the Screen Templates tab, a different Screen is displayed instead of the intended Template.

## Steps to Reproduce:

1. Save a desired Screen as a Template.
2. Navigate to the Screen Templates tab. (My Templates and Public Templates)
3. Within the Ellipsis menu, Select Edit Template.
4. Observe that a different Screen is shown, not the selected Screen Template.

## Expected Behavior:
Users should be able to navigate to the Screen Templates tab, select a saved Screen Template, and be presented with the correct Template for editing. 

## Solution
- Implement Editing functionality by importing the Screen Template manifest and Creating a hidden Screen

## How to Test
- Verify that the Screen Template you observe matches exactly with the one initially created from the Screen.
- Follow the same steps for My Templates and Public Templates


ci:next
ci:deploy

## Related Tickets & Packages
- Link to any related FOUR tickets, PRDs, or packages

## Code Review Checklist
